### PR TITLE
Translation zh

### PR DESCRIPTION
**Pull request by xiaod-dev**
submit new Chinese translation files

### CODE REVIEW

File: zh/composite/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+```java
+var orcMessage = new Messenger().messageFromOrcs();
+orcMessage.print(); // Where there is a whip there is a way.
+var elfMessage = new Messenger().messageFromElves();
+elfMessage.print(); // Much wind pours from your mouth.
+```
+
+## 类图
+
+![alt text](../../Composite/etc/composite.urm.png "Composite class diagram")
```
Comment (ohbus) [inline zh/composite/README.md:156]: I believe there is some problem with the path the reason why this png is not coming.

File: zh/state/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    mammoth.observe();
+    
+    // The mammoth gets angry!
+    // The mammoth is furious!
+    // The mammoth calms down.
+    // The mammoth is calm and peaceful.
+```
+
+## 类图
+![alt text](../../state/etc/state_1.png "State")
```
Comment (ohbus) [inline zh/state/README.md:137]: I believe the path problem might also exist here.

Please check.

File: zh/template-method/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+```java
+    var thief = new HalflingThief(new HitAndRunMethod());
+    thief.steal();
+    thief.changeMethod(new SubtleMethod());
+    thief.steal();
+```
+
+## 类图
+![alt text](../../template-method/etc/template-method_1.png "Template Method")
```
Comment (ohbus) [inline zh/template-method/README.md:121]: The path seems to be broken.

**Review by ohbus — CHANGES_REQUESTED** [review]
Please confirm if the given links are broken or not which are mentioned in the comments.

File: zh/state/README.md (hunk 2 above)
Comment (xiaod-dev) [reply zh/state/README.md]: The png name seems changes. Problem solved.

File: zh/template-method/README.md (hunk 3 above)
Comment (xiaod-dev) [reply zh/template-method/README.md]: The png name seems changes. Problem solved.

File: zh/composite/README.md (hunk 1 above)
Comment (ohbus) [reply zh/composite/README.md:156]: @xiaod-dev I believe this has not yet been fixed?

**Review by ohbus — CHANGES_REQUESTED** [review]

File: zh/composite/README.md (hunk 1 above)
Comment (xiaod-dev) [reply zh/composite/README.md]: emmm....The path works fine in my Typora. I update 'Composite' into 'composite'. Please check it again.

**Review by ohbus — APPROVED** [review]
LGTM

Thanks @xiaod-dev for this awesome contribution.

**Pull request merged**